lib/serviceBusMessage.ts

```diff
@@ -955,8 +955,11 @@ export class ServiceBusMessage implements ReceivedMessage {
     };
     if (options) {
       error = {
-        condition: options.deadletterReason,
-        description: options.deadLetterErrorDescription
+        condition: Constants.deadLetterName,
```

Comment:
I would suggest to set this condition in line 953 and remove the description on line 954

Reply:
Great catch!